frontend/src/components/common/mobile-sidebar/mobile-sidebar.module.scss

```diff
@@ -13,7 +13,7 @@
   opacity: 0;
   transition: 0.3s;
 
-  @media screen and (min-width: 1300px) {
+  @media screen and (min-width: 1301px) {
     display: none;
```

Comment:
i suggest you move breakpoint values to assets/css/breakpoints, if you have time

Reply:
@foxt451 we have other widths values.
I don't think we should to expand it now.
To many scss files, which using custom media query values, now.